catalog/task/update-infra-deployments/0.1/update-infra-deployments.yaml

```diff
@@ -0,0 +1,333 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1a"
```

Comment:
I think this should just be 0.1 seeing as we are storing it as version 0.1

Reply:
yes. not sure why I added `a`

fixed